tests/acceptance/features/coreApiWebdavLocks/exclusiveLocks.feature

```diff
@@ -66,7 +66,7 @@ Feature: there can be only one exclusive lock on a resource
       | new      | shared     |
       | new      | exclusive  |
 
-  @skipOnOcV10 @issue-34358 @notToImplementOnOCIS
+  @notToImplementOnOCIS
```

Comment:
you have removed @notToImplementOCIS test, also remove here (?)

Reply:
We have separate PR to remove `@notToImplementOnOcis`